requirements.txt

```diff
@@ -49,6 +49,9 @@ git+git://github.com/PyCQA/pylint.git@2561f539d60a3563d6507e7a22e226fb10b58210
 # the tests.  With this here, you can run the tests with `py.test` from the base directory.
 pytest
 
+# Static type checking
+mypy
```

Comment:
It also makes our docker image smaller for when you're training a model, or serving predictions.